From: Markus Koschany <apo@debian.org>
Date: Sat, 3 Oct 2020 02:56:25 +0200
Subject: platform-libs.junit5

junit5 has been packaged now but I'm not sure if we actually need it here.
---
 platform/libs.junit5/nbproject/project.properties | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/platform/libs.junit5/nbproject/project.properties b/platform/libs.junit5/nbproject/project.properties
index 4a04140..e5627a5 100644
--- a/platform/libs.junit5/nbproject/project.properties
+++ b/platform/libs.junit5/nbproject/project.properties
@@ -16,9 +16,6 @@
 # under the License.
 
 is.autoload=true
-release.external/junit-jupiter-api-5.6.0.jar=modules/ext/junit-jupiter-api-5.6.0.jar
-release.external/junit-jupiter-params-5.6.0.jar=modules/ext/junit-jupiter-params-5.6.0.jar
-release.external/junit-jupiter-engine-5.6.0.jar=modules/ext/junit-jupiter-engine-5.6.0.jar
 nbm.target.cluster=platform
 sigtest.skip.check=true
 sigtest.fail.on.error=false
